portal-impl/src/com/liferay/portal/tools/HypersonicLoader.java

```diff
@@ -95,10 +95,13 @@ public HypersonicLoader(
 		// See LEP-2927. Appending ;shutdown=true to the database connection URL
 		// guarantees that ${databaseName}.log is purged.
 
+		String connectionProperties =
```

Comment:
After our review process, we should forward this code to core-infra team so they can check this part of the code